### PR TITLE
[SR-24] Only run appropriate specs for spree-reports

### PR DESCRIPTION
**Pull request by mbajur**

### CODE REVIEW

File: backend/app/queries/spree/admin/reports/average_order_values_query.rb [hunk 1]
```diff
@@ -33,8 +33,15 @@ def group_by_date
         end
 
         def apply_date_filters(scope)
-          scope = scope.where('completed_at >= ?', opts[:completed_at_min]) if opts[:completed_at_min]
-          scope = scope.where('completed_at <= ?', opts[:completed_at_max]) if opts[:completed_at_max]
+          if opts[:completed_at_min]
+            min = Time.zone.parse(opts[:completed_at_min]).beginning_of_day
```
Comment (przemosk) [inline backend/app/queries/spree/admin/reports/average_order_values_query.rb]: shouldn't we use `Time.current` ?

Comment (mbajur) [reply backend/app/queries/spree/admin/reports/average_order_values_query.rb]: nope, we can't do `Time.current.parse('...')`

File: backend/app/queries/spree/admin/reports/average_order_values_query.rb [hunk 2]
```diff
@@ -33,8 +33,15 @@ def group_by_date
         end
 
         def apply_date_filters(scope)
-          scope = scope.where('completed_at >= ?', opts[:completed_at_min]) if opts[:completed_at_min]
-          scope = scope.where('completed_at <= ?', opts[:completed_at_max]) if opts[:completed_at_max]
+          if opts[:completed_at_min]
```
Comment (piotrleniec-spark) [inline backend/app/queries/spree/admin/reports/average_order_values_query.rb]: Why do we need these changes? They're out of the scope of `SR-24` task.

Comment (mbajur) [reply backend/app/queries/spree/admin/reports/average_order_values_query.rb]: because not having that change breaks the specs on CI (timezone issues)

**Pull request merged**